src/safeposix/syscalls/fs_calls.rs

```diff
@@ -814,6 +823,48 @@ impl Cage {
         ret
     }
 
+    pub fn sbrk(&self, brk: u32) -> i32 {
+        let mut vmmap = self.vmmap.write();
```

Comment:
same as above

Reply:
realistically sbrk doesnt need a function in fs_calls at all, it can just be a helper function in mem.rs